gradle.properties

```diff
@@ -1,11 +1 @@
 version=4.18.0
-#
-# main dependencies
-#
-swarm-extension-sdk.version=4.18.0
```

Comment:
This change means that we need to update the updater here: https://github.com/hivemq/hello-world-extensions-version-updater/blob/39e8320b19c5a004cf3e9411a44b9b7ae7564a7e/update-hello-world-swarm-extension.sh#L9

Reply:
Created a separate card: https://hivemq.kanbanize.com/ctrl_board/41/cards/17046/details/